app/javascript/app/components/focus-trap-proxy.js

```diff
@@ -1,13 +1,11 @@
-/* eslint-disable */
-import FocusTrap from 'focus-trap';
-/* eslint-enable */
+import createFocusTrap from 'focus-trap';
 
 function FocusTrapProxy() {
   const focusables = [];
   let activated = [];
 
   return function makeTrap(el, options = {}) {
-    const ownTrap = new FocusTrap(el, options);
+    const ownTrap = createFocusTrap(el, options);
```

Comment:
Upgrading Sinon had me encounter an issue with how the `focus-trap` library was stubbed to assume the usage of a constructor. I also identified this in #3938, but the usage here does not appear to be correct. The default export of `focus-trap` is intended to be called as a factory function, not as a constructor.

See documentation for the version of the package used in the project:

https://github.com/davidtheclark/focus-trap/tree/2.3.0#focustrap--createfocustrapelement-createoptions